code/modules/surgery/surgery_step.dm

```diff
@@ -12,6 +12,8 @@
 
 /datum/surgery_step/proc/try_op(mob/user, mob/living/target, target_zone, obj/item/tool, datum/surgery/surgery, try_to_fail = FALSE)
 	var/success = FALSE
+	if(user == target)
+		time = time*3
```

Comment:
```suggestion
		time *= 3
```

Reply:
oh I didn't know you could write it like that